docs/install-audit-log-filter.md

```diff
@@ -0,0 +1,42 @@
+# Install the Audit Log Filter
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The `plugin_dir` system variable defines the plugin library location. If needed, at server startup, set the `plugin_dir` variable.
+
+When upgrading a MySQL installation, plugins are not automatically upgraded. You may need to manually load certain functions or reinstall the plugin after the MySQL upgrade.
+
+In the `share` directory, locate the `audit_log_filter_linux_install.sql `script.
+
+Prior to 8.0.34, use this command:
```

Comment:
We don't have this plugin for versions less than 8.0.34 so no need to mention how it worked before.